STM32F405RGTx/Core/Inc/LED_thread.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "util.hpp"
+
+class LEDThread {
```

Comment:
Is there a reason we're not doing the LEDStatus here like in MC? I think it's best to keep it consistent across the boards. For something as common as and LED Thread, maybe we can somehow implement this in the util library and you would just have to add an LED pin and port into the initialize function

Reply:
I would be down to put it in util, but I was honestly just lazy tbh lol.

Reply:
@Nushaab-Syed @ren-andy Can you add this as an item somewhere for an improvement in the future? An LED thread could definitely be put in the util library. Maybe you can think of other stuff we could add to the util library. Besides that, this convo is resolved.

Reply:
> 

Would this be suitable as a G5.5 or G6 improvement? I'll add it to the planning package accordingly.